### PR TITLE
Fix js-api version in adf-cli

### PR DESCRIPTION
**Pull request by MichalKinas**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [x] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**

Now js-api version in adf-cli will be aligned with js-api version in main package.json

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Review by nikita-web-ua — APPROVED** [review]

File: .github/workflows/upstream-js.yml [hunk 1]
```diff
@@ -113,10 +113,13 @@ jobs:
...
 
           migrateDependenciesJS() {
-              echo "Update JS dependencies to: ${PACKAGE_VERSION_JS}"
-              echo "Calling migration JS"
-              npx nx migrate @alfresco/js-api@${PACKAGE_VERSION_JS}
-              echo "Migration JS done"
+            echo "Update JS dependencies to: ${PACKAGE_VERSION_JS}"
+            echo "Calling migration JS"
+            npx nx migrate @alfresco/js-api@${PACKAGE_VERSION_JS}
+            cd lib/cli
```
Comment (DenysVuika) [inline .github/workflows/upstream-js.yml:119]: you can reduce the code if taking into brackets

```sh
(cd lib/cli && npx nx migrate @alfresco/js-api@${PACKAGE_VERSION_JS})
```

in that case you don't need "cd ../.."

File: lib/cli/package.json [hunk 2]
```diff
@@ -20,7 +20,7 @@
     "dist": "rm -rf ../../dist/libs/cli && npm run build && cp -R ./bin ../../dist/libs/cli && cp -R ./resources ../../dist/libs/cli && cp -R ./templates ../../dist/libs/cli && cp ./package.json ../../dist/libs/cli"
   },
   "dependencies": {
-    "@alfresco/js-api": "^6.1.0",
+    "@alfresco/js-api": "6.2.0-982",
```
Comment (DenysVuika) [inline lib/cli/package.json:23]: should have caret otherwise may fail again

File: .github/workflows/upstream-js.yml (hunk 1 above)
Comment (MichalKinas) [reply .github/workflows/upstream-js.yml]: Changed

File: lib/cli/package.json (hunk 2 above)
Comment (MichalKinas) [reply lib/cli/package.json]: Added

**Review by DenysVuika — APPROVED** [review]

**Pull request merged**